packages/math-input/src/index.ts

```diff
@@ -31,6 +31,16 @@ export {MobileKeypad} from "./components/keypad";
 // Unwrapped v2 keypad for desktop
 export {default as DesktopKeypad} from "./components/keypad";
 
+// For strings used in MathQuill and the keypad
+export {strings} from "./strings";
```

Comment:
@jeremywiebe I'm not happy about this, I would much prefer that we build and export this separately, so that it's not part of the bundle. For example I could import it like so:

```
import strings from "@khanacademy/math-input/strings";
```

And not from the main export itself. Do you have a sense of how to best configure something like that? If not, I can experiment with making that happen in rollup.

Reply:
Just to follow up on this, I got this working! There are now multiple exports.